.travis.yml

```diff
@@ -9,20 +18,24 @@ env:
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
-    - php: 7.0
-      env: DB=PGSQL PHPUNIT_TEST=1
+      env: DB=MYSQL INSTALLER_VERSION=4.4.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
+      env: DB=PGSQL INSTALLER_VERSION=4.5.x-dev PHPUNIT_COVERAGE_TEST=1 PDO=1
     - php: 7.2
-      env: DB=MYSQL PHPUNIT_TEST=1
+      env: DB=PGSQL INSTALLER_VERSION=4.6.x-dev PHPUNIT_TEST=1
+    - php: 7.3
+      env: DB=MYSQL INSTALLER_VERSION=4.6.x-dev PHPUNIT_TEST=1
+    - php: 7.4
+      env: DB=MYSQL INSTALLER_VERSION=4.x-dev PHPUNIT_TEST=1
 
 before_script:
   - phpenv rehash
   - phpenv config-rm xdebug.ini
 
-  - if [[ $DB == PGSQL ]]; then composer require --prefer-dist --no-update silverstripe/postgresql 2.1.x-dev; fi
-  - composer require --no-update silverstripe/installer:4.x-dev
+  - composer require --no-update silverstripe/installer:$INSTALLER_VERSION
+  # Fix for running phpunit 5 on php 7.4+
+  - composer require --no-update sminnee/phpunit-mock-objects:^3
```

Comment:
this can be combined with the above line, not much point keeping them separate

Reply:
Could do, though we've got the message `Fix for running phpunit 5 on php 7.4+` above it, which Robbie asked me to put in on a similar travis PR so that future developers can know what it's for and when it's safe to remove.  Prefer to keep as is